packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -669,15 +684,18 @@ $nav-popover-padding               : spacing(x2);
         flex-direction: column;
     }
 }
-    .c-nav-list-item {
-        margin-bottom: 0;
 
-        // TODO: MAKE THIS NOT USE FLOATS
-        // global modifier for list items horizontally aligned
-        .c-nav--global & {
-            @include media('>=mid') {
-                float: left;
-            }
+    .c-nav-popoverList--twoColumns {
+        @include media('>=mid') {
+            column-count: 2;
+        }
+    }
+
+    // TODO: MAKE THIS NOT USE FLOATS
+    // global modifier for list items horizontally aligned
+    .c-nav-list-item--horisontallyAlignedOnWiderView {
```

Comment:
Could use the `AboveMid` naming convention here?

```suggestion
    .c-nav-list-item--horisontallyAlignedAboveMid {
```

Reply:
I will include this change to the next PR